### DISCUSSION TITLE
How can I use metrics from the SqlClient Instrumentation library without adding tracing?

### DISCUSSION THREAD

**Discussion by kharabasz** [open-telemetry/opentelemetry-dotnet#5458] [General]
Some libraries like the AspNetCore Instrumentation provide the ability to add metrics and tracing separately:

```
otel
    .WithMetrics(mp => mp.AddAspNetCoreInstrumentation())
    .WithTracing(tp => tp.AddAspNetCoreInstrumentation());
```

How can I use metrics from the SqlClient Instrumentation library without adding tracing? 

```
otel
    .WithMetrics(mp => mp.???())
    .WithTracing(tp => tp.AddSqlClientInstrumentation());
```

**Comment by cijothomas** [comment]
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.SqlClient does not support metrics today. Only traces are supported in SqlClient instrumentation!
